External/FEXCore/Source/Interface/Core/OpcodeDispatcher/Vector.cpp

```diff
@@ -435,9 +435,10 @@ void OpDispatchBuilder::MOVQOp(OpcodeArgs) {
   // This instruction is a bit special that if the destination is a register then it'll ZEXT the 64bit source to 128bit
   if (Op->Dest.IsGPR()) {
     const auto gpr = Op->Dest.Data.GPR.GPR;
-    _StoreContext(FPRClass, 8, offsetof(FEXCore::Core::CPUState, xmm[gpr - FEXCore::X86State::REG_XMM_0][0]), Src);
```

Comment:
Unrelated to this PR but we should probably modify how this code behaves in the future.
We have vector ops that can move a register with zext, we should just use that zext 64-bit to 128bit instead of the nasty here.

After this PR though, I don't want to change op behaviour if possible.